test/e2e/openshift.cnv.workflow.sh

```diff
@@ -12,16 +12,14 @@ export FIRST_SECONDARY_NIC=${FIRST_SECONDARY_NIC:-ens7}
 export SECOND_SECONDARY_NIC=${SECOND_SECONDARY_NIC:-ens8}
 export TIMEOUT=${TIMEOUT:-60m}
 
-if [ ! -f  $SSH ]; then
-    cat << EOF > ${SSH}
+cat << EOF > ${SSH}
```

Comment:
Please split this into as many PR as many independent change syou introduce. There are at least 4 of them. It is important for all branches, but especially for stable ones.

Reply:
I can close one eye when there is code refactoring piggy-backed with other changes. But not for unrelated changes.